packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -45,6 +45,10 @@ const propTypes = {
    * Whether the textarea can be auto-resized vertically. _(Will be ignored if size attribute is set to "full".)_
    */
   isAutoResizable: PropTypes.bool,
+  /**
+   * Whether the form is incomplete. Form must also be required.
```

Comment:
Let's update our descriptions for isIncomplete to say:
```suggestion
   * Whether the textarea displays as Incomplete. Use when no value has been provided. _(usage note: `required` must also be set show display)_
```


\
Also update isInvalid to match and say:
```
   * Whether the textarea displays as Invalid. Use when value does not meet validation pattern.
```

Reply:
Updated the `isInvalid` and `isComplete` prop descriptions for all: https://github.com/cerner/terra-core/pull/2670/commits/f3184124d28bce916fe72b76f0f19fa0aea8c0fd